### ISSUE TITLE
Connection to notebook kernel lost upon exceptions

### ISSUE THREAD

**Issue by jdoepfert** [pauldmccarthy/fsleyes#33]
Upon throwing an exception in th fsleyes notebook, the connection to the notebook kerne seems to be lost, as no more cells can be executed:

1) Everything works as expected:
![image](https://user-images.githubusercontent.com/7914340/66985159-793bf080-f0bc-11e9-9686-c22e36752d0b.png)

2) Exception due to wrong call:
![image](https://user-images.githubusercontent.com/7914340/66985191-8bb62a00-f0bc-11e9-93b7-38dc4f66d694.png)

3) No more cells can be executed:
![image](https://user-images.githubusercontent.com/7914340/66985248-a7b9cb80-f0bc-11e9-922c-8aa28bc536ab.png)

Code:
```
displayCtx.getOpts(overlayList[0])

displayCtx.getOpts()

displayCtx.getOpts(overlayList[0])

print(1)
```

**Comment by pauldmccarthy** [comment]
Yes, it looks like this is the case; I have figured out a workaround (hard-coding [`stop_on_error=False`](https://github.com/ipython/ipykernel/blob/master/ipykernel/kernelbase.py#L528) ), but don't quite understand the mechanics yet, as `stop_on_error` defaults to True even in normal notebook execution. WIll keep digging.

**Comment by jdoepfert** [comment]
thanks

**Comment by pauldmccarthy** [comment]
Figured it out, will be fixed on the master branch within the next couple of hours, and in the next version (although I don't know when that will be).

**Comment by jdoepfert** [comment]
awesome 👍

**Comment by pauldmccarthy** [comment]
This should be resolved as of 0.32.0 - please get back to me if you haveany more problems with it.

**Comment by jdoepfert** [comment]
great. will test as soon as the standalone package is available
